homeassistant/components/media_player/plex.py

```diff
@@ -192,6 +200,19 @@ def update_devices():
             client.set_availability(client.machine_identifier
                                     in available_client_ids)
 
+            if config.get(CONF_REMOVE_UNAVAILABLE_CLIENTS) \
```

Comment:
For the future: you could have considered a guard clause here to prevent indentations.

```python
if not config.get(CONF_REMOVE_UNAVAILABLE_CLIENTS) or client.available:
    continue

interval = …
```

Reply:
Yes that would have made sense